### PR TITLE
Update SQL

### PR DESCRIPTION
**Pull request by lesv**
1. Use filtering to set variables.
2. Update README

I was unable to get SQL proxy working so we can’t really do IT testing
yet.

### CODE REVIEW

**Comment by lesv** [comment]
Tim - PTAL - I'm going to merge.

File: appengine/cloudsql/README.md [hunk 1]
```diff
@@ -1,13 +1,25 @@
 # Cloud SQL sample for Google App Engine
 This sample demonstrates how to use [Cloud SQL](https://cloud.google.com/sql/) on Google App Engine
+
 ## Setup
-Before you can run or deploy the sample, you will need to create a [Cloud SQL instance](https://cloud.google.com/sql/docs/create-instance)
-1. Create a new user and database for the application. The easiest way to do this is via the [Google Developers Console](https://console.developers.google.com/project/_/sql/instances/example-instance2/access-control/users). Alternatively, you can use MySQL tools such as the command line client or workbench.
+Before you can run or deploy the sample, you will need to create a [Cloud SQL instance)](https://cloud.google.com/sql/docs/create-instance)
+
+1. Create a new user and database for the application. The easiest way to do this is via the [Google
+Developers Console](https://console.developers.google.com/sql/instances). Alternatively, you can use MySQL tools such as the command line client or workbench.
```
Comment (tswast) [inline appengine/cloudsql/README.md]: Should update the link to point to the cloud console.

File: appengine/cloudsql/README.md [hunk 2]
```diff
@@ -1,13 +1,25 @@
...
+2. Change the root password (under Access Control) and / or create a new user / password.
```
Comment (tswast) [inline appengine/cloudsql/README.md]: Can't create dbs or users from the cloud console with v2 instances yet, AFAIK.

File: appengine/cloudsql/README.md [hunk 3]
```diff
@@ -1,13 +1,25 @@
...
+3. Create a Database (under Databases) (or use MySQL with `gcloud sql connect <instance> --user=root`)
+4. Note the **Instance connection name** under Overview > properties
+(It will look like project:instance for v1 or project:region:zone for v2)
+
+## Deploying
+
+`$ mvn clean appengine:deploy -DINSTANCE_CONNECTION_NAME="instanceConnectionName" -Duser=root
```
Comment (tswast) [inline appengine/cloudsql/README.md]: Okay, but I prefer the 4-space indent or

```
code blocks with three ```
```

Also, it's a bit funny to see a property name that isn't `camelCase` such as `INSTANCE_CONNCETION_NAME` but probably fine.

File: appengine/cloudsql/README.md (hunk 2 above)
Comment (lesv) [inline appengine/cloudsql/README.md]: Actually, I can.  Maybe not yet deployed?

File: appengine/cloudsql/README.md (hunk 1 above)
Comment (lesv) [inline appengine/cloudsql/README.md]: What's wrong with the link?

Comment (tswast) [inline appengine/cloudsql/README.md]: Should be https://console.cloud.google.com/sql/instances Remember console.developers.google.com just has the API manager now and no cloud stuff.

**Comment by lesv** [comment]
Fixed in #452

**Pull request merged**